allauth/account/adapter.py

```diff
@@ -676,6 +679,9 @@ def get_login_stages(self):
             ret.append("allauth.mfa.stages.AuthenticateStage")
         return ret
 
+    def send_notification_mail(self, template_prefix, user, context):
+        self.send_mail(template_prefix, user.email, context)
```

Comment:
I think we need an overall switch to prevent emitting notifications: `ACCOUNT_EMAIL_NOTIFICATIONS = True | False`.